changelogs/fragments/51661-fixing-azure-mysql-parameters.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - fixed issues with passing parameters while updating existing mysql
```

Comment:
I would prefix this message with the module name, e.g. `azure_rm_mysqlserver - fixed issues....`